src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java

```diff
@@ -235,11 +236,12 @@ public void onBindHeaderViewHolder(HeaderViewHolder viewHolder, int position) {
         txt = context.getString(R.string.profile_shared_chats);
         break;
       case ItemData.TYPE_PRIMARY_SETTING:
-        if(itemDataContact!=null && itemDataContact.isVerified()) {
-          txt = context.getString(R.string.verified_contact);
+        long lastSeen = (itemDataContact!=null? itemDataContact.getLastSeen() : 0);
+        if(lastSeen==0) {
```

Comment:
```suggestion
        if (lastSeen == 0) {
```

i think, this is the style we are mostly using for new code.

(existing code may use different styles, if pulled from some others sources, in general, we do not change the style to to retain the option of easy diffs on updates, same for tabs :)